### DISCUSSION TITLE
Thank you for this project!

### DISCUSSION THREAD

**Discussion by code-ethan** [audacious-media-player/audacious#1537] [General]
I apologize if this is too off-topic or anything (feel free to remove it if so), but I just wanted to extend a heartfelt thank you to everyone who works on Audacious and makes it a reality. With streaming services taking over the way we consume media (mostly for the worse), I find myself turning back to collecting albums as FLAC files on my local hard drive like the olden days. And needless to say, Audacious handles the task like a champ. No other modern apps come remotely close to the ease of use and beautiful simplicity that Audacious provides.

So once again, thank you so much to everyone who works on this project! Hope this note finds you well, and happy new year. Cheers!

**Comment by radioactiveman** [comment]
Thanks for the kind words Ethan, much appreciated. Happy New Year to you too.

**Comment by atomGit** [comment]
> With streaming services taking over the way we consume media (mostly for the worse), I find myself turning back to collecting albums as FLAC files on my local hard drive like the olden days.

ditto!

i also appreciate the cleanliness of the UI and the options it offers, both on the backend and the UI - very nice piece of work

**Comment by pferdefreund** [comment]
It is for me the best player i ever heared. Good work.
Thank you
